ruff.toml

```diff
@@ -0,0 +1,35 @@
+line-length = 120
```

Comment:
Since we already have a top-level `pyproject.toml` file, we could move all these settings over there instead of `ruff.toml` like 

https://github.com/apache/airflow/blob/d03b9a76914babef23b84acdb6061b0b93bdc2e3/pyproject.toml#L29

Reply:
We didn't do it in Astro SDK as it is a mono-repo

Reply:
fixed it